vumi_http_proxy/servicemaker.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
```

Comment:
This crept in from #9, but you can sort it out once you've merged that branch by then merging develop into this branch.